airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -190,6 +191,11 @@ class Events(IterableStream):
     data_field = "events"
     page_size = EVENT_ROWS_LIMIT
 
+    @staticmethod
+    def _check_email_is_valid(email):
+        email_regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b"
```

Comment:
maybe we are filtering emails too aggressively here -- can you be more specific about what constitutes an "invalid" email? should an email with an emoji succeed? what about email with weird chars e.g ™£¢

Reply:
> should an email with an emoji succeed? what about email with weird chars e.g ™£¢

Well, we cannot help those who are looking for trouble.